### PR TITLE
Update appVersion for 1.3.7 release

### PR DESCRIPTION
**Pull request by rishabh6788**
Signed-off-by: Rishabh Singh <sngri@amazon.com>

### Description
Update appVersion for 1.3.7 release
 
### Issues Resolved
https://github.com/opensearch-project/opensearch-build/issues/2742
 
### Check List
- [x] Commits are signed per the DCO using --signoff

For any changes to files within Helm chart directories:
- [ ] Helm chart version bumped
- [ ] Helm chart `CHANGELOG.md` updated to reflect change

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/helm-charts/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: charts/opensearch-dashboards/CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +12,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 ---
+## [1.9.2]
```
Comment (peterzhuamazon) [inline charts/opensearch-dashboards/CHANGELOG.md:15]: Every new app version must be a minor bump.

File: charts/opensearch/CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 ---
+## [1.16.2]
```
Comment (peterzhuamazon) [inline charts/opensearch/CHANGELOG.md:15]: Same as above.

**Review by peterzhuamazon — CHANGES_REQUESTED** [review]

**Comment by peterzhuamazon** [comment]
@rishabh6788 please rebase on main as well. Thanks.

File: charts/opensearch-dashboards/Chart.yaml [hunk 3]
```diff
@@ -15,13 +15,13 @@ type: application
...
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.9.1
+version: 1.10.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.3.6"
+appVersion: "1.3.7"
```
Comment (gaiksaya) [inline charts/opensearch-dashboards/Chart.yaml:24]: AFAIK we do not change the app version right? @peterzhuamazon

Comment (peterzhuamazon) [reply charts/opensearch-dashboards/Chart.yaml:24]: App Version is the docker image version.
Change it.

Comment (peterzhuamazon) [reply charts/opensearch-dashboards/Chart.yaml:24]: Since 1.3.7 is out to public right now, thanks.

**Review by peterzhuamazon — APPROVED** [review]

**Review by zelinh — APPROVED** [review]

**Pull request merged**